### PR TITLE
TCT-ColBERT-V2 Reproduce

### PR DESCRIPTION
**Pull request by jacklin64**
@MXueguang the index and query are in the tuna1/scratch/sclin/tct_colbert-v2

### CODE REVIEW

File: docs/experiments-tct_colbert-v2.md
Comment (lintool) [inline docs/experiments-tct_colbert-v2.md:26]: Can you clarify this? which number is from the paper, which is reprod?

e.g., "figures from the paper are in parentheses"

**Review by MXueguang — APPROVED** [review]
lgtm, I'll manage indexes/queries after this PR merged.

**Pull request merged**
